Paskab012/cartix-web1
# Cartix signup form

Hello Reviewer ✋🏽 
## Description of changes made ♻️

> This PR adds authentication components to the project. The design of `signin`, `signup`, and `forgot password` forms were created with form input validation.
The found error should be fixed as quickly as the bug is reported so far.
Now the inputs can be filled, error messages can appear if once the required input was not filled by the user.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](git@github.com:Ossix-Technologies/2u-webapp.git)
- Check out this branch by `branch here`
- Run `yarn or npm install to install the project dependencies
- Run `npm run start:cartix` to start the development server

#### Checklist


#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| 1. [Given Figma design](https://www.figma.com/file/jP4w1VZFvauBSiByZTrY4c/AFR-SG-MAP-V2?node-id=314%3A5456) | 2. Implemented: `Run ====> npm run start:cartix` |
|
